server/models/event.ts

```
@@ -160,6 +160,18 @@ const EventSchema = new Schema(
       required: false,
       default: null,
     },
+    PreScreening: { 
```

ESLint

[error] Delete `·`

(prettier/prettier)